### PR TITLE
Implementing react-alerts

### PR DESCRIPTION
**Pull request by andrewoh531**

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Alert from '..'
+
+ReactDOM.render(
+  <Alert type="error" title="Let's get started">
+    {['Completing your provider check will allow you to update provider details and access secure data. '
```
Comment (jameslnewell) [inline example/index.js]: Don't need the `{}s` or `[]`s here. This should work fine:

```javascript
<Alert type="error" title="Let's get started">
  Completing your provider check will allow you to update provider details and access secure data. <a href="#">Verify my providers.</a>
</Alert>
```

**Comment by jameslnewell** [comment]
Can you please rm+ignore `example/index.bundled.js`.

Otherwise looks good!

**Comment by jameslnewell** [comment]
@lauriejones @eastwood any further thoughts?

**Comment by lauriejones** [comment]
Looks good to me!

**Pull request merged**